app/screens/navigation.ts

```diff
@@ -523,6 +523,15 @@ export async function popTopScreen(screenId?: AvailableScreens) {
     }
 }
 
+export async function popTo(screenId: AvailableScreens) {
```

Comment:
How is this different from the function above?

Reply:
So the above function `popTopScreen` just pops the top-most screen, like going back one step. Here, we want to pop all the components. The given component `popTo` clears everything above the screen you pass in, making that passed screen the top screen.